packages/atti-components/src/components/Breadcrumb/components/BreadcrumbElement.js

```diff
@@ -1,22 +1,25 @@
 // @noSnapshot
 import React from 'react'
+import PropTypes from "prop-types";
 import Link from '../../Link'
-import { ElementType } from '../types'
+import CurrentElement from "../styles/CurrentElement";
 
 /**
  * A breadcrumb element.
  */
-const BreadcrumbElement = ({ element }) => (
-  <Link href={element.url} onClick={element.onClick}>
-    {element.name}
-  </Link>
-)
+const BreadcrumbElement = (props) => {
+  const { children, url } = props
+
+  return url ? <Link href={url}>
+      {children}
+    </Link>
+    :
+    <CurrentElement {...props}/>
+}
 
 BreadcrumbElement.propTypes = {
-  /**
-   * Breadcrumb element.
-   */
-  element: ElementType.isRequired,
+  /** Element's url */
+  url: PropTypes.string,
```

Comment:
Since this PR is bringing a breaking-change, let's change the API to make the component standardized. Probably better to change `url` to `href` to be consitent. And it will reduce enable you to reduce the code base as well:
```jsx
const BreadcrumbElement = props => {
  return props.href ? <Link {...props}> : <CurrentElement {...props}/>
}
```

Reply:
This is a good idea, thanks Alexandre 👍

Reply:
Done